docs/Installation-Guide.rst

```diff
@@ -179,26 +224,25 @@ Then run the following commands:
 
 **Note**: glibc >= 2.14 is required.
 
-OSX
-^^^
-
-Install **gcc** and **Open MPI** first:
+macOS
```

Comment:
OSX different vs macOS?

Reply:
it's like saying Linux OS
OSX is a version
macOS in the OS

Reply:
To be more precise:
>macOS (/ˌmækoʊˈɛs/; previously Mac OS X and later OS X) is the current series of Unix-based graphical operating systems developed and marketed by Apple Inc. ...

[wiki1](https://en.wikipedia.org/wiki/MacOS), [wiki2](https://en.wikipedia.org/wiki/OSX_(disambiguation))
So previously it was **OS X**, but now **macOS** is a right name.